src/data/matrix/basic.lean

```diff
@@ -878,6 +882,13 @@ by { ext, refl }
 
 end transpose
```

Comment:
You should copy this entire `transpose` namespace (from line 830 to here), and make a version of each of these lemmas for `conj_transpose`

Reply:
However; you should be able to prove them with `by rw [conj_transpose, the_transpose_lemma, the_map_lemma]` if you use the `map` definition I suggest, rather than proving them from scratch.

Reply:
I am going to add this block in#8289

Reply:
You should add it in this PR instead. It's much better to make a PR that adds one feature thoroughly than it is to make a PR that adds a lot of features half-heartedly.

Reply:
How about close this PR and add things into #8289? This would be more convenient for me. @eric-wieser

Reply:
No, I strongly recommend pausing all work on #8289 and adding all the things related to `conj_transpose` in this PR. Once this PR is done and merged, `git` can update the other PR to match.

Reply:
sure